<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileListWindow</class>
 <widget class="QMainWindow" name="FileListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow desktop</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="fileListView"/>
    </item>
    <item>
     <widget class="QListView" name="samplesListView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomButtonsLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLocation">
    <property name="title">
     <string>Location</string>
    </property>
    <addaction name="aLocationAssets"/>
    <addaction name="aLocationDocuments"/>
   </widget>
   <addaction name="menuLocation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aLocationAssets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Assets</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="aLocationDocuments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Documents</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>aLocationAssets</sender>
   <signal>triggered()</signal>
   <receiver>FileListWindow</receiver>
   <slot>SelectAssets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aLocationDocuments</sender>
   <signal>triggered()</signal>
   <receiver>FileListWindow</receiver>
   <slot>SelectDocuments()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectAssets()</slot>
  <slot>SelectDocuments()</slot>
 </slots>
</ui>
